src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -70,6 +70,11 @@
 public final class StartsDaemonITCase {
 
     /**
+     * Rultor repo.
+     */
+    private static final String RULTOR = "yegor256/rultor";
```

Comment:
@xupyprmv this can just be a local variable in `deprecatesDefaultImage`, it's only used there :) No need to add this kind of noise. Putting this kind of thing outside of the function and into a constant only makes sense if we really need Javadoc on it, but I think here we're both clear on the meaning of the variable :)